samples/storage_csm_powerflex_v2100.yaml

```diff
@@ -224,6 +224,14 @@ spec:
           # image: Defines karavi-topology image. This shouldn't be changed
           # Allowed values: string
           image: dellemc/csm-topology:v1.8.0
+          # certificate: certificate for cert/private-key pair -- please add cert here to use custom certificates
```

Comment:
based on internal files, these values need to be base64 encoded right?
If so- it would be a good idea to add that comment to the samples files

Reply:
yeah I can do that